### PR TITLE
Move black check to github actions

### PR DESCRIPTION
**Pull request by camilamaia**
## Description
Move black check from CircleCI to GitHub actions:

https://github.com/marketplace/actions/black-code-formatter

Closes #136

### CODE REVIEW

File: scanapi/__init__.py [hunk 1]
```diff
@@ -53,12 +53,12 @@ def scan(spec_path, output_path, config_path, reporter, template, log_level):
     try:
         api_spec = load_config_file(spec_path)
     except FileNotFoundError as e:
-        error_message = f"Could not find API spec file: {spec_path}. {str(e)}"
-        logger.error(error_message)
+        error_msg = f"Could not find API spec file: {spec_path}. {str(e)}"
```
Comment (camilamaia) [inline scanapi/__init__.py:56]: I did this change only to test black lint :D

**Comment by camilamaia** [comment]
@barbosa it didnt run. Any guess?

**Comment by camilamaia** [comment]
> @barbosa it didnt run. Any guess?

I found the problem 😅

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+name: Lint check
+jobs:
+  black:
+    name: Lint check
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Black Code Formatter"
+        uses: "lgeiger/black-action@v1.0.1"
+        with:
+          args: ". --check"
```
Comment (camilamaia) [inline .github/workflows/lint.yml:11]: I am not sure this is getting the proper path 🤔 

image

File: scanapi/__init__.py (hunk 1 above)
Comment (barbosa) [reply scanapi/__init__.py:56]: was this supposed to be caught?

Comment (camilamaia) [reply scanapi/__init__.py:56]: It is not a lint error, but the file should be checked

Comment (camilamaia) [reply scanapi/__init__.py:56]: Let me add an error

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+on: pull_request
+name: Lint check
+jobs:
+  black:
+    name: Lint check
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Black Code Formatter"
```
Comment (barbosa) [inline .github/workflows/lint.yml:8]:
```suggestion
      - uses: actions/checkout@v1
      - name: "Black Code Formatter"
```

**Pull request merged**